gemfiles/rails_5.0.gemfile

```diff
@@ -3,6 +3,5 @@
 source "https://rubygems.org"
 
 gem "rails", "~> 5.0.0"
-gem "sprockets", "3.7.2"
```

Comment:
Let's keep it here, I want to test with Sprockets 3.x specifically

Reply:
I don't know why this has been removed. I will add it to the `Appraisals`

Reply:
ah, probably that's the answer, these files may be overwritten by appraisal